system/include/webgpu/webgpu.h

```diff
@@ -918,10 +961,18 @@ typedef struct WGPUBlendState {
 } WGPUBlendState;
 
 typedef struct WGPUCompilationInfo {
+    WGPUChainedStruct const * nextInChain;
     uint32_t messageCount;
     WGPUCompilationMessage const * messages;
 } WGPUCompilationInfo;
 
+typedef struct WGPUComputePassDescriptor {
+    WGPUChainedStruct const * nextInChain;
+    char const * label;
+    uint32_t timestampWriteCount;
+    WGPUComputePassTimestampWrite const * timestampWrites;
```

Comment:
This is a new addition to the API and needs to be implemented (or at least TODO'd)... but it will also disappear if you make the generator changes, so let's just do that for now. Implementing this can be a task for later.